.travis.yml

```diff
@@ -0,0 +1,53 @@
+language: python
```

Comment:
I've been thinking about whether not to cut travis; as it stands now, it does basically the same thing as azure, except it's configured to also run weekly cron jobs (these also need to be configured in the settings menu over at travis-ci.org !) with stable, numpy-dev and astropy-dev.

It probably is good to have it as a backup for ~~the inevitable future collapse of Microsoft~~ Azure outages, though.